client/src/__tests__/errorBoundary.test.tsx

```diff
@@ -0,0 +1,36 @@
+import "jest-dom/extend-expect";
+import "react-testing-library/cleanup-after-each";
+
+import React from "react";
+import { render } from "react-testing-library";
+import ErrorBoundary from "../ErrorBoundary";
+
+jest.mock("../Header", () => ({
+  HEIGHT: 50
+}));
+jest.mock("../basePath", () => "");
+
+beforeEach(() => {
+  // when the error's thrown a bunch of console.errors are called even though
+  // the error boundary handles the error. This makes the test output noisy,
+  // so we'll mock out console.error
+  jest.spyOn(console, "error");
+  (console.error as any).mockImplementation(() => undefined);
+});
+
+afterEach(() => {
+  (console.error as any).mockRestore();
```

Comment:
I'm a fan of [jest `clearMocks` option](https://jestjs.io/docs/en/configuration.html#clearmocks-boolean). Would propose to use that instead of manually restoring it.

Reply:
Let's do it!